ui/app/AppLayouts/Profile/views/EnsTermsAndConditionsView.qml

```diff
@@ -302,11 +302,16 @@ Item {
             StatusBaseText {
                 text: qsTr("Agree to <a href=\"#\">Terms of name registration.</a> I understand that my wallet address will be publicly connected to my username.")
                 anchors.left: termsAndConditionsCheckbox.right
+                anchors.leftMargin: Style.current.halfPadding
                 anchors.right: parent.right
                 wrapMode: Text.WordWrap
-                anchors.top: termsAndConditionsCheckbox.top
+                anchors.verticalCenter: termsAndConditionsCheckbox.verticalCenter
                 onLinkActivated: popup.open()
                 color: Theme.palette.directColor1
+                TapHandler {
```

Comment:
Didn't know this was a thing!

Reply:
Yeah this thing doesn't conflict with a MouseArea 🙂